src/blog/components/customErrorMessage/CustomErrorMessage.tsx

```diff
@@ -0,0 +1,59 @@
+import Button from "src/components/buttons/Button";
+import LinkButton from "src/components/linkButton/LinkButton";
+import Text from "src/components/text/Text";
+import { ILink, LinkType } from "studio/lib/payloads/navigation";
+import styles from "./customErrorMessage.module.css";
+
+export const homeLink = {
+  _key: "return-home",
+  _type: "link",
+  linkTitle: "Return to home",
+  linkType: LinkType.Internal,
+  internalLink: {
+    _ref: "/",
+  },
+};
+
+export const studioLink = {
+  _key: "go-to-studio",
+  _type: "link",
+  linkTitle: "Go to studio",
+  linkType: LinkType.Internal,
+  internalLink: { _ref: "/studio" },
+};
+
+interface CustomErrorMessageProps {
+  title: string;
+  body: string;
+  button?: {
+    label: string;
+    onClick: () => void;
+  };
+  link?: ILink;
+}
+
+const CustomErrorMessage = ({
+  title,
+  body,
+  button,
+  link,
+}: CustomErrorMessageProps) => {
+  return (
+    <section className={styles.wrapper}>
```

Comment:
might want to give this a test to check if we want it or not. 🤔 
`role="alert" aria-live="assertive"`

Reply:
Not sure of it fits this description

> `aria-live="assertive"` should only be used for time-sensitive/critical notifications that absolutely require the user's immediate attention. Generally, a change to an assertive live region will interrupt any announcement a screen reader is currently making. As such, it can be extremely annoying and disruptive and should only be used sparingly.

(https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#live_regions)

Reply:
I think it does fit.

That way the focus is on the error message upon loading and not on "skip to main" and header. 💡

Reply:
I guess it depends on how the component is used. If it's a full page error it probably fits, but if it's only concerning a small not-so-important component it might be annoying?

Reply:
Regardless if it's a full page or a box, it's an error with a warning status, if it were an information box, it'd have aria-live="assertive".

In this case the ErrorMessage works as a full page.

Reply:
ops no, I meant to write polite 😂